UnityProject/Assets/Scripts/Weapons/WeaponNetworkActions.cs

```diff
@@ -155,12 +155,15 @@ public void CmdUnloadWeapon(GameObject gunObject)
 					if(isWeapon && weaponStats != null &&
 					weaponStats.hitSoundSettings == SoundItemSettings.Both)
 					{
-						SoundManager.PlayNetworkedAtPos(integrity.soundOnHit, gameObject.WorldPosServer(), Random.Range(0.9f, 1.1f), sourceObj: gameObject);
+						// JESTER
+						// SoundManager.PlayNetworkedAtPos(integrity.soundOnHit, gameObject.WorldPosServer(), Random.Range(0.9f, 1.1f), sourceObj: gameObject);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)